src/Packetery/Module/HookHandler.php

```diff
@@ -76,6 +86,12 @@ public function updateOrder( $wcOrderId ): void {
 			$this->orderRepository->delete( (int) $wcOrderId );
 		}
 
+		try {
```

Comment:
Try catch prosím pryč. Metodu pojmenuj podle hooku, tedy woocommerceUpdateOrder.

Reply:
Je to pryc.